ocrd/ocrd/decorators.py

```diff
@@ -27,9 +32,7 @@ def ocrd_cli_wrap_processor(processorClass, ocrd_tool=None, mets=None, working_d
         print(msg)
```

Comment:
Why not use logging here?

Reply:
I thought there was a reason but there really isn't. I'll log them as errors.

Reply:
The reason was that otherwise it couldn't be tested easily. But it wasn't the most significant test anyway.